exchanges/okx/okx_websocket.go

```diff
@@ -1365,9 +1378,6 @@ func (ok *Okx) WsPlaceMultipleOrder(args []PlaceOrderRequestParam) ([]OrderData,
 			arg.OrderType == OkxOrderFOK || arg.OrderType == OkxOrderIOC) {
 			return nil, fmt.Errorf("invalid order price for %s order types", arg.OrderType)
 		}
-		if arg.QuantityType != "base_ccy" && arg.QuantityType != "quote_ccy" {
```

Comment:
Why was this removed? When I made [this](https://github.com/thrasher-corp/gocryptotrader/pull/1005#discussion_r1014923175) comment, I meant for you to add a valid test scenario
If this validation is no longer required, then what about all other instances of it?